doc/subsystems/networking/usbnet_setup.rst

```diff
@@ -0,0 +1,79 @@
+.. _usb_device_networking_setup:
+
+USB Device Networking
+#####################
+
+This page describes how to set up networking between a Linux host
+and a Zephyr application running on USB supported devices. A sample
+application from Zephyr source distribution is run on one of
+supported boards. The board is connected to Linux host using USB cable
+and provides an Ethernet interface to the host.
+
+Basic Setup
+***********
+
+To communicate with Zephyr application over newly created Ethernet
```

Comment:
Change to:

    To communicate with the Zephyr application over a newly created Ethernet
    interface, we need to assign IP addresses and set up a routing table.  After
    plugging a USB cable from the board to the Linux host, the ``cdc_ether`` driver
    registers a new Ethernet device with a provided MAC address.